src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletControllerTest.java

```diff
@@ -365,4 +366,68 @@ void shouldThrowAnExceptionWhenWalletIdEqualsZero() throws Exception {
         //then
         result.andExpect(status().isNoContent());
     }
+
+    @Test
+    void shouldReturnStatusOkWhenFindWalletByIdCorrectly() throws Exception {
```

Comment:
``shouldReturnStatusOkWhenWalletHasBeenFound()`` sounds much better, but I think that It doesn't really matter.

Reply:
Of course, I corrected name of method.